recipes/PyAV/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "av" %}
```

Comment:
This is another occurrence of issue ( https://github.com/conda-forge/staged-recipes/issues/1073 ).